CHANGELOG.md

```diff
@@ -3,6 +3,25 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## v3.1.0 - 2025-03-04
+
+- Move `txp2gene` to `reference_genome_options` in schema as it is required by `kb_python` and `alevin` ([434](https://github.com/nf-core/scrnaseq/pull/434))
+- Fix of additional path splitting for `txp2gene` ([433](https://github.com/nf-core/scrnaseq/pull/433))
+- Add a checker so that `--fb_reference` does not break the pipeline in case `ab` files are not used in `cellranger multi` sub-workflow.
+- Fix concatenation of multiple samples into the combined output AnnData ([416](https://github.com/nf-core/scrnaseq/pull/416))
+- Make sure STARsolo velocity output is added to the combined output AnnData, if `star_feature = 'Gene Velocyto'` ([417](https://github.com/nf-core/scrnaseq/pull/417))
+- Update cellbender module to latest nf-core version ([#419](https://github.com/nf-core/scrnaseq/pull/419/))
+- Add profile for gpu processes ([#419](https://github.com/nf-core/scrnaseq/pull/419/))
+- Update example usage command in README with valid reference genome parameter ([#339](https://github.com/nf-core/scrnaseq/issues/339))
+- Removed `--kb_filter` parameter. Kallisto filtering is triggered by default and can be turned off with `ext.args` ([#421](https://github.com/nf-core/scrnaseq/issues/421))
+- Document better that `cellbender` is used for empty drops calling and not the `emptydrops` method (([#420](https://github.com/nf-core/scrnaseq/issues/420)))
+- Add `--limitBAMsortRAM` to STARsolo alignment, to make sure BAM sorting memory scales with the task memory ([#430](https://github.com/nf-core/scrnaseq/pull/430))
+- Replace local modules for simpleaf, SIMPLEAF_INDEX and SIMPLEAF_QUANT, with their central modules from nf-core/modules, and update simpleaf subworkflows accordingly.
+- Update documents related to `simpleaf`, `alevin`, `salmon`, and `alevin-fry` for consistency.
+- Rename the default aligner from `alevin` to `simpleaf` for consistency.
```

Comment:
this sounds like a breaking change tbh. so, highlight it in the beginning and depending how strict you are with semver, make a major version bump.

Reply:
What exactly? Renaming the aligner? That could be addressed with backwards-compatible arguments mentioned below. 

I think the definition of a "breaking" change in a pipeline is almost philosophical, because Semver is intended for APIs, but what's the API of a pipeline? All input parameters? Nextflow config files that change `ext.args` for a specific process? The output folder structure? 

IMO SemVer was a poor choice for pipelines, but that's a different discussion.

Reply:
we recently made our versioning philosophy a more concrete: https://nf-co.re/docs/guidelines/pipelines/requirements/semantic_versioning#semantic-versioning-criteria but we are not too strict with it.